lib/node_modules/@stdlib/types/index.d.ts

```diff
@@ -593,6 +593,85 @@ declare module '@stdlib/types/array' {
 	* const x: Collection<number> = [ 1, 2, 3 ];
 	*/
 	type Collection<T = any> = Array<T> | TypedArray | ArrayLike<T>; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+	/**
+	* Type mapping for the following array data types:
+	*
+	* -   `float64`: double-precision floating-point numbers (IEEE 754)
+	* -   `float32`: single-precision floating-point numbers (IEEE 754)
```

Comment:
Are these lists necessary? I am thinking mainly from a future maintenance perspective in case of drift (e.g., the addition of `float16` and the comment is not updated. If this helps with IDE support, then fine, but not sure otherwise.

Reply:
Let's remove them!